src/main/java/org/wso2/greg/plugin/workspace/WSO2GregWorkspace.java

```diff
@@ -0,0 +1,243 @@
+                    UISupport.showErrorMessage(String.format("Unable to create Project because of %s exception with "
+                                    + "\"%s\" message", e.getClass().getName(),
+                            e.getMessage()));
+                    return;
+                }
+                List<RestService> services =
+                        ResourceImporterWorker.importServices(selectionResult, project);
+                if (services != null && !services.isEmpty()) {
+                    UISupport.select(services.get(0));
+                } else {
+                    workspace.removeProject(project);
+                }
+
+
+            }
+        }
+    }
+
+    private void saveLoginProfileData(String host, String port, String userName, String tenantDomain) throws GregReadyPluginException {
+
+        File profileDataFile;
+
+        profileDataFile = new File(System.getProperty("java.io.tmpdir") + File.separator +
+                ResourceConstants.LOGIN_PROFILE_FILE_PREFIX +
+                ResourceConstants.LOGIN_PROFILE_FILE_SUFFIX);
+
+        if (profileDataFile.exists()) {
+            profileDataFile.delete();
+        }
+        try (
+            BufferedWriter out = new BufferedWriter(new FileWriter(profileDataFile));
+        )
+        {
+            profileDataFile.createNewFile();
+
+            String data = ResourceConstants.URLS.HOST_NAME + ":" + host + "\n" +
+                    ResourceConstants.URLS.PORT + ":" + port + "\n" +
+                    ResourceConstants.URLS.USER_NAME + ":" + userName + "\n" +
+                    ResourceConstants.TENANT_DOMAIN + ":" + tenantDomain + "\n";
+
+            out.write(data);
+            out.flush();
+
+        } catch (IOException ioEx) {
+            log.error("IO Exception occured while saving login profile", ioEx);
+            throw new GregReadyPluginException("IO Exception occurred while saving login profile", ioEx);
+        }
+    }
+
+    private void setProfileValuesToUI(XFormDialog dialog) {
+
+        File loginPrfileSettingsFile = new File(System.getProperty("java.io.tmpdir") + File.separator +
+                ResourceConstants.LOGIN_PROFILE_FILE_PREFIX + ResourceConstants.LOGIN_PROFILE_FILE_SUFFIX);
+
+        Map profileMap = new HashMap();
+
+        if (loginPrfileSettingsFile.exists()) {
+
+            try (
+                 FileInputStream fis = new FileInputStream(loginPrfileSettingsFile);
+                 BufferedReader br = new BufferedReader(new InputStreamReader(fis));
+            ){
+                String line;
+
+                while ((line = br.readLine()) != null) {
+                    try {
+                        profileMap.put(line.split(":")[0], line.split(":")[1]);
+                    }catch(PatternSyntaxException patEx){
+                        log.error("Pattern syntax error in setting profile data"+line,patEx);
+                    }catch (ArrayIndexOutOfBoundsException arrEx){
```

Comment:
Formatting issues.
1. There should be a space in between ) and catch block
2. There should be a space in between ) and { characters